app/src/main/java/com/concordia/mcga/fragments/BottomSheetBuildingInfoFragment.java

```diff
@@ -81,10 +84,20 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
         setupBottomSheetBehavior();
         overrideBottomSheetCallBack();
         setupListAdapter();
+
         return view;
     }
 
-
+    public void updateBottomSheet(){
+        // TEMPORARY
+        clear();
+        if (buildingName.equals("H")) {
+            displayHBuildingAssociations();
+        } else if (buildingName.equals("JM")) {
+            displayMBBuildingAssociations();
+        }
+        collapse();
+    }
```

Comment:
Actually, I don't even know how I would populate the list without the name of the building